python/private/pypi/deps.bzl

```diff
@@ -130,11 +131,32 @@ def pypi_deps():
     Fetch dependencies these rules depend on. Workspaces that use the pip_parse rule can call this.
     """
     for (name, url, sha256) in _RULE_DEPS:
-        maybe(
-            http_archive,
-            name,
-            url = url,
-            sha256 = sha256,
-            type = "zip",
-            build_file_content = _GENERIC_WHEEL,
-        )
+        if name in ["pypi__installer", "pypi__packaging"]:
+            maybe(
+                http_archive,
+                name,
+                url = url,
+                sha256 = sha256,
+                type = "zip",
+                build_file_content = _GENERIC_WHEEL,
+            )
+        else:
+            maybe(
+                whl_archive,
+                name,
+                urls = [url],
+                sha256 = sha256,
+                requirement = name.split("_")[-1],
+                repo = "pypi_",
```

Comment:
I thought it would be nice to dogfood the library, but we need to have a reference to the default (or some other) python interpreter in order to do this.

In `bzlmod` it is possible to do this, but in `WORKSPACE` it is not, so we may need to leave the `deps.bzl` as is.